nosekatreport/plugin.py

```diff
@@ -1180,10 +1242,6 @@ def end(cls, passed=None, message=None, traceback=None):
                 fail_message = '\033[91m\033[1m %s \033[0m' %(fail_message)
                 raise TestFailed(fail_message)
```

Comment:
This will cause a traceback, ass TestFailed class was commented out.
Uncomment TestFailed class on ln #45